src/main/java/us/ihmc/zed/library/ZEDJavaAPINativeLibrary.java

```diff
@@ -41,10 +50,60 @@ public NativeLibraryWithDependencies getLibraryWithDependencies(OperatingSystem
     private static boolean loaded = false;
 
     public static boolean load() {
-        if (!loaded) {
-            ZEDJavaAPINativeLibrary lib = new ZEDJavaAPINativeLibrary();
-            loaded = NativeLibraryLoader.loadLibrary(lib);
+        String installedZEDSDKVersion = getZEDSDKCMakePackageVersion();
+
+        int installedZEDSDKMajorVersion = 0;
+        int installedZEDSDKMinorVersion = 0;
+
+        if (installedZEDSDKVersion != null) {
+            String[] installedZEDSDKVersionParts = installedZEDSDKVersion.split("\\.");
+
+            try {
+                installedZEDSDKMajorVersion = Integer.parseInt(installedZEDSDKVersionParts[0]);
+                installedZEDSDKMinorVersion = Integer.parseInt(installedZEDSDKVersionParts[1]);
+            } catch (NumberFormatException e) {
+                // Ignore
+            }
         }
+
+        if (installedZEDSDKMajorVersion >= ZED_SDK_COMPATIBILITY_MAJOR_VERSION
+                && installedZEDSDKMinorVersion >= ZED_SDK_COMPATIBILITY_MINOR_VERSION) {
```

Comment:
There's a slight flaw in this logic. If current version is 4.2 and user has 5.0 installed it'll print error. 

Perhaps something like 
```
if (majorVersion > ZED_SDK_MAJOR_VERSION 
    || (majorVersion == ZED_SDK_MAJOR_VERSION && minorVersion >= ZED_SDK_MINOR_VERSION))
```